src/libmongoc/tests/test-mongoc-collection.c

```diff
@@ -5630,6 +5630,62 @@ test_create_indexes_with_opts_commitQuorum_post44 (void *unused)
    mongoc_client_destroy (client);
 }
 
+void
+test_insert_one_reports_id (void)
+{
+   mongoc_client_t *client = test_framework_new_default_client ();
+   mongoc_collection_t *coll = get_test_collection (client, "test_insert_reports_id");
+   bson_error_t error;
+   bool ok;
+
+   // Test inserting one document reports the inserted ID.
+   {
+      bson_t *doc = tmp_bson ("{'_id': 'foo'}");
+      bson_t reply;
+      ok = mongoc_collection_insert_one (coll, doc, NULL /* opts */, &reply, &error);
+      ASSERT_OR_PRINT (ok, error);
+      // Check that `reply` contains the inserted ID in an array `ids`.
```

Comment:
```suggestion
      // Check that `reply` contains the inserted ID.
```

Suggestion also applies to other comments in this function.